src/db/mysql/models.rs

```diff
@@ -867,12 +854,6 @@ struct UserCollectionsResult {
     modified: i64,
 }
 
-#[derive(Debug, QueryableByName)]
-struct Count {
-    #[sql_type = "BigInt"]
-    count: i64,
-}
-
 /// Formats a BSO for UPDATEs
 #[derive(AsChangeset)]
```

Comment:
You can kill UpdateBSO along with this change as well

Reply:
Done